src/PluginsSystem/version.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
+  "version": "1.1-preview",
```

Comment:
Should we start these off at 0.1-preview? Don't really see the need to rush to 1.0 :)

Reply:
@trgibeau What do you think about the version number?